Procfile

```diff
@@ -1 +1 @@
-web: Run serve --env production --port $PORT --hostname 0.0.0.0
+web: Run serve --env production --port $PORT --hostname 0.0.0.0 --config:postgresql.url $DATABASE_URL
```

Comment:
Is the Profile specific to Heroku? Not sure how app bootstrapping looks in ECS

Reply:
I think so. We probably need talk to Myung